client/js/components/user/CustomerSettings/CustomerSettings.vue

```diff
@@ -430,6 +521,120 @@ export default {
       }
     },
 
+    resetAccessibilityExplanationSettings () {
+      this.customer.accessibilityExplanation = this.customerList[this.currentCustomerId].attributes.xplanning
+    },
+
+    resetImprintSettings () {
+      this.customer.imprint = this.customerList[this.currentCustomerId].attributes.imprint
+    },
+
+    resetDataProtectionSettings () {
+      this.customer.dataProtection = this.customerList[this.currentCustomerId].attributes.dataProtection
+    },
+
+    resetOverviewDescriptionInSimpleLanguageSettings () {
+      this.customer.overviewDescriptionInSimpleLanguage = this.customerList[this.currentCustomerId].attributes.overviewDescriptionInSimpleLanguage
+    },
+
+    resetTermsOfUseSettings () {
+      this.customer.termsOfUse = this.customerList[this.currentCustomerId].attributes.termsOfUse
+    },
+
+    resetXplanningSettings () {
+      this.customer.xplanning = this.customerList[this.currentCustomerId].attributes.xplanning
+    },
+
+    saveAccessibilityExplanationSettings () {
```

Comment:
I believe the save methods can also be replaced by a single generic method with the property to be updated as parameter

Reply:
Does this work? My idea was to pass the property to be reset as a string, e.g. `resetProperty('imprint')` instead of `resetProperty(customer.imprint)`

Reply:
Sorry, the comment refers to the conversation above (but the commit is the same)

Reply:
Actually, the question applies to `saveSettings`, too :see_no_evil:

Reply:
yes, its working this way

Reply:
I tested `resetProperty` and it does not work for me. If you do it like this:
`resetProperty(customer.imprint)`
what is passed to `resetProperty` as a parameter is the value of `customer.imprint`. This code will then not work:
`this.customer[property] = currentCustomer.attributes[property]`
because what you need is the key of the property (in this case, 'imprint').

Reply:
I must second @hwiem . It can't work the way its build right now.
If it works, then by accident.

Reply:
okay, I will change it than

Reply:
I understand the point here